pages/dkp/kommander/1.3/introducing-kommander/index.md

```diff
@@ -18,12 +18,14 @@ Kommander was developed to address the broad issues caused by Kubernetes "cluste
 | **Features**                                                                                      | **Benefits**                                                                                                                                                                                                                                             |
 | ------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | **Operations Dashboard**                                                                          | Provides instant visibility and operational efficiency into Kubernetes cluster landscape, helping an organization understand resources and utilization.                                                                                                  |
-| [**Centralized Monitoring, Logging, and Alerting**](/dkp/kommander/1.3/centralized-monitoring) | Enables central monitoring and alerting of issues arising in operational and application infrastructure and deep cross-cluster logging for easier troubleshooting. As issues are detected, they are resolved before they escalate, saving valuable time. |
+| [**Centralized Monitoring and Alerting**](/dkp/kommander/1.3/centralized-monitoring) | Provides enhanced visibility and control at the enterprise level, with comprehensive monitoring across all clusters. |
```

Comment:
I removed logging references here because we don't actually provide centralized logging 😅